app/models/transport_order.rb

```diff
@@ -29,7 +29,7 @@ class TransportOrder < ApplicationRecord
   belongs_to :operation
   belongs_to :transporter
   belongs_to :contract
-
+  has_many :transport_order_items
   def bid
```

Comment:
@nathnael @dawitd this is not a good idea to pull association detail like this. I can see this is for use in [TO list view](https://github.com/ndrmc/cats-web/blob/3f0060981739e1923607a1f0ff6a30f0bcc889e1/app/views/transport_orders/show.html.erb#L138). I suggest to fetch related entities using 'include' method. Using methods like this is going to trigger separate trip to the DB for each TO but if we include it in the original request then it will be only two trips - one to fetch all TOs the other to fetch all FDPs for all TOs in the previous query (using **in** clause for SQL) - [Look at Gift Certificate for more on this](https://github.com/ndrmc/cats-web/blob/3f0060981739e1923607a1f0ff6a30f0bcc889e1/app/controllers/gift_certificates_controller.rb#L9)